packages/logger-pretty/lib/LoggerPretty.ts

```diff
@@ -1,5 +1,5 @@
-import { inspect } from 'util';
 import { Logger } from '@comunica/types';
+import objectInspect = require('object-inspect');
```

Comment:
Is the `require` really needed here?
It might be possible to replace it with `import * as objectInspect from 'object-inspect'`

Reply:
Indeed. That's much better. I have updated the PR.